springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/scanners/channels/ChannelsScanner.java

```diff
@@ -10,5 +11,10 @@ public interface ChannelsScanner {
     /**
      * @return A mapping of channel names to their respective channel object for a given protocol.
      */
-    Map<String, ChannelItem> scan();
+    Map<String, ChannelObject> scanChannels();
+
+    /**
+     * @return A mapping of operation names to their respective operation object for a given protocol.
+     */
+    Map<String, Operation> scanOperations();
```

Comment:
Now the `ChannelScanner` scans for both `channels` and `operations`. Those components are decoupled in the new AsyncAPI v3 spec, so maybe would make sense to have a standalone `OperationsScanner`.

I think would be good to define it before the 1.0 release.

Reply:
Good question. I believe they most likely come in pairs, still I am in favour of splitting this into two interfaces.

Reply:
I'm preparing a PR with this change. It's a bit more work than expected 😄

Reply:
@timonback please take a look to https://github.com/springwolf/springwolf-core/pull/562 it became larger than expected